index.html

```diff
@@ -45,23 +45,169 @@ <h2 class="primary-header">Glad to see you!</h2>
+          </li>
+          <li>
+            <img src="./src/images/Counter.svg" alt="dote icon">
+          </li>
+          <li>
+            <p>2015</p>
+          </li>
+        </ul>
+        <p class="details">
+          Experimental content creation feature that allows users to add to an existing story over the course of a day
+          without spamming their friends.
+        </p>
+        <div class="tags">
+          <p class="tag-item">html</p>
+          <p class="tag-item">Ruby on rails</p>
+          <p class="tag-item">css</p>
+          <p class="tag-item">javaScript</p>
+        </div>
+        <button class="btn">See Project</button>
+      </div>
+      <img src="./src/images/multi-post.png" alt="">
+    </section>
+    <section class="wrapper card">
+      <img src="./src/images/facebook.png" alt="">
+      <div class="description">
+        <h2 class="secondary-header">
+          Facebook 360
+        </h2>
+        <ul class="counter">
+          <li>
+            <p>FACEBOOK</p>
+          </li>
+          <li>
+            <img src="./src/images/Counter.svg" alt="dote icon">
+          </li>
+          <li>
+            <p>Full Stack Dev</p>
+          </li>
+          <li>
+            <img src="./src/images/Counter.svg" alt="dote icon">
+          </li>
+          <li>
+            <p>2015</p>
+          </li>
+        </ul>
+        <p class="details">
+          Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360
+          photos and videos on Gear VR.
+        </p>
+        <div class="tags">
+          <p class="tag-item">html</p>
+          <p class="tag-item">Ruby on rails</p>
+          <p class="tag-item">css</p>
+          <p class="tag-item">javaScript</p>
+        </div>
+        <button class="btn">See Project</button>
+      </div>
+    </section>
+
+    <section class="wrapper card">
+      <div class="description">
+        <h2 class="secondary-header">
+          Uber Navigation
+        </h2>
+        <ul class="counter">
+          <li>
+            <p>Uber</p>
+          </li>
+          <li>
+            <img src="./src/images/Counter.svg" alt="dote icon">
+          </li>
+          <li>
+            <p>Lead Developer</p>
+          </li>
+          <li>
+            <img src="./src/images/Counter.svg" alt="dote icon">
+          </li>
+          <li>
+            <p>2018</p>
+          </li>
+        </ul>
+        <p class="details">
+          A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer:
+          your car.
+        </p>
+        <div class="tags">
+          <p class="tag-item">html</p>
+          <p class="tag-item">Ruby on rails</p>
+          <p class="tag-item">css</p>
+          <p class="tag-item">javaScript</p>
+        </div>
+        <button class="btn">See Project</button>
+      </div>
+      <img src="./src/images/uber.png" alt="">
+    </section>
```

Comment:
- [ ] You are almost there! It would be great to add the About section, as this is required by the activity:

<img width="441" alt="Screen Shot 2022-08-18 at 7 56 49 PM" src="https://user-images.githubusercontent.com/83717285/185520067-c92a232c-c633-4056-8642-0fd08b20611b.png">